src/components/OrderHistory/OrderHistoryText.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { useRouter } from 'next/navigation';
+
+import querystring from 'querystring';
+import { ViewOrderButton, ArrowIcon } from './styles'; // Adjust the import path as necessary
+
+function formatDate(isoString: string) {
+  const options: Intl.DateTimeFormatOptions = {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+  };
+  const date = new Date(isoString);
+  return date.toLocaleDateString('en-US', options);
+}
+
+interface OrderDetailsProps {
+  date: string;
+  orderNumber: string;
+  status: string; // Define more statuses if needed
+}
+
+export default function OrderDetails(props: OrderDetailsProps) {
+  const { date, orderNumber, status } = props;
+  const router = useRouter();
+
+  const viewOrder = (orderID: string) => {
+    const queryString = querystring.stringify({ orderID });
+    router.push(`/orderPage?${queryString}`);
+  };
+
+  return (
+    <div
+      style={{
+        display: 'flex',
+        justifyContent: 'space-between',
+        alignItems: 'center',
+        marginBottom: '20px',
+      }}
+    >
+      <div>
+        <div>{formatDate(date)}</div>
```

Comment:
too many divs! if its text, it should be surrounded by a styled text componet from the global folder